src/java.base/share/classes/java/lang/Enum.java

```diff
@@ -172,9 +172,9 @@ public final boolean equals(Object other) {
      * The hash code of this enumeration constant.
      *
      * @implNote Once initialized, the field value does not change.
-     * Hotspot's identity hash code generation also never returns zero
-     * as the identity hash code. This allows to treat zero as the marker
-     * for the un-initialized value for both {#link @Stable} and the lazy
+     * HotSpot's identity hash code generation also never returns zero
+     * as the identity hash code. This makes zero a convenient marker
+     * for the un-initialized value for both {@link @Stable} and the lazy
```

Comment:
I'm sorry, but `@Stable` won't work with `@link`. `@Stable` is not a class, interface, member name, etc. See https://docs.oracle.com/en/java/javase/20/docs/specs/javadoc/doc-comment-spec.html#references

Reply:
The \@implNote is more appropriate for an internal comment. It is not needed to be in the published javadoc; its only useful to someone viewing the source code.

Reply:
> The @implNote is more appropriate for an internal comment. It is not needed to be in the published javadoc; its only useful to someone viewing the source code.

Is it a comment to me or to the author? If it's the former, then I note, that if one bothers to mark up a comment, that markup should be valid.

Reply:
I would change this to be an non-javadoc comment.  (Change /** to /*)
An there is no need for javadoc \implNote or \@link javadoc markup.
There is very little value in being able to generate javadoc for --private.
(In part because most private implementations are not documented to javadoc standards).

Reply:
Yes, it is a bit strange to see @implNote here as this is a private field that isn't going to show up in the javadoc. The javadoc for non-transient fields in Serializable classes does show up in the Serialized Form page but this Enum which is special.